database/factories/OrganizerFactory.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use App\Models\Organizer;
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Organizer>          
+ */
+class OrganizerFactory extends Factory
+{
+    protected $model = Organizer::class;
+    protected $qt;
+    protected static $names_map = [
+        'Greenpeace',
+        'Kick Out Zwarte Piet',
+        'Nederland Wordt Beter',
+        'Woonprotest',
+
+    ];
+
+    protected static $status_map = [
+        'PENDING',
+        'APPROVED',
+        'REJECTED',
+        'PUBLISHED'        
+    ] ;
+    
+    
+    public function definition(): array
+    {   
+        # This number is the same in the count parameter in the seeder file
+        $tot_elements = $this->count;
```

Comment:
Is this used?

Reply:
I think the better function to use in line 48 would be $faker->numberBetween(1,$tot_elements), where $tot_elements is the number of instances to be created, passed as parameter in the seed file, in the count() method.

But, I  got a exception when I use it, the exception is below transcripted. In the Stackoverflow there was explanation about this error sometimes occur with small datasets. So I changed to function to
$faker->randomElement(), but I will search other ways to solve it in the future and then use the variable $tot_elements. 


Exception : " FAILED  Tests\Feature\RouteTest > available routes wit…  OverflowException   
  Maximum retries of 10000 reached without finding a unique value
  at vendor/fakerphp/faker/src/Faker/UniqueGenerator.php:80"